go.mod

```diff
@@ -187,7 +187,7 @@ require (
 	github.com/tidwall/pretty v1.2.0 // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
-	go.uber.org/multierr v1.9.0 // indirect
```

Comment:
It was upgraded in a previous PR, that was eventually not using `multierr`. I just ran `go mod tidy` and it auto-downgraded, so... I don't know?

Reply:
@shlomi-noach might be because the direct dependency was dropped? A `go get -u go.uber.org/multierr` probably would add this back correctly (it should be there, so good catch @GuptaManan100).

Reply:
This is fixed.